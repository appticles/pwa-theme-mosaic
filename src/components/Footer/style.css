.posts-carousel-container .footer {
  position: absolute;
  bottom: 0;
  height: unset;
  display: flex;
  justify-content: space-between;
  width: 100%;
  background: #fafafa;
  padding: 5px;
  border-top: solid 1px #dddddd;
  z-index: 1000; }
