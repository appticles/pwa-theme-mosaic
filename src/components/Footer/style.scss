@import "../../utilities/variables";

.posts-carousel-container .footer {
  position: absolute;
  bottom: 0;
  height: unset;
  display: flex;
  justify-content: space-between;
  width: 100%;
  background: $footer-bg-color;
  padding: $base-spacing-xs;
  border-top: solid 1px $border-color;
}
