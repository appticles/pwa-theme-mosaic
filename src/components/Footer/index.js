import React from 'react';
import { Link } from 'react-router-dom';
import { Icon } from 'semantic-ui-react';

import './style.css';

const Footer = () => (
  <div className="footer">
    <Link to="/">
      <Icon name="grid layout" size="big" />
    </Link>
  </div>
);

export default Footer;
