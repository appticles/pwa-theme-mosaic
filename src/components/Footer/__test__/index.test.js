import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Footer from '../index';

describe('Footer', () => {
  it('should render correctly', () => {
    const output = shallow(<Footer />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });

  it('should render correctly with title and page', () => {
    const output = shallow(<Footer title="Footer title" page={2} pageLabel="Page" />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
