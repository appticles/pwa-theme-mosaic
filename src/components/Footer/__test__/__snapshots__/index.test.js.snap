// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<div
  className="footer"
>
  <Link
    replace={false}
    to="/"
  >
    <Icon
      as="i"
      name="home"
      size="big"
    />
  </Link>
</div>
`;
